nautobot/dcim/models/device_components.py

```diff
@@ -773,6 +778,87 @@ def parent(self):
         return self.device
 
 
+@extras_features(
+    "custom_fields",
+    "custom_links",
+    "custom_validators",
+    "export_templates",
+    "graphql",
+    "relationships",
+    "statuses",
+    "webhooks",
+)
+class InterfaceRedundancyGroup(PrimaryModel, ConfigContextModel, StatusModel):  # pylint: disable=too-many-ancestors
+    """
+    A collection of Interfaces that supply a redundancy group for protocols like HSRP/VRRP.
+    """
+
+    name = models.CharField(max_length=100, unique=True)
+    slug = AutoSlugField(populate_from="name")
```

Comment:
Idk about still having `slug` here.  Since `name` is already unique, can we just safely delete `slug` from the model so we do not have to do the extra work when 2.0 comes around?

Reply:
`slug` is now removed.